### PR TITLE
WFLY-13463 Issues in readme for microprofile-metrics QS

### PR DESCRIPTION
**Pull request by xstefank**
https://issues.redhat.com/browse/WFLY-13463

### CODE REVIEW

**Comment by xstefank** [comment]
@nziakova can you also take a look at this one, please?

File: microprofile-metrics/README.adoc [hunk 1]
```diff
@@ -85,19 +99,19 @@ Now we need to add the following dependencies:
 
 [source,xml]
 ----
-<!-- Import the MicroProfile Metrics API, we use provided scope as the API is included in WildFly -->
+<!-- Import the MicroProfile Metrics API, we use provided scope as the API is included in the server -->
```
Comment (nziakova) [inline microprofile-metrics/README.adoc:102]: Will these also be updated in pom.xml?

**Comment by nziakova** [comment]
@xstefank Thank you for the fixes. Readme looks good to me, but what about the comments in pom.xml?

**Comment by xstefank** [comment]
@nziakova I think that @emmartins will rebrand those as part of EAP rebranding. @emmartins am I correct?

**Comment by nziakova** [comment]
Ah right, thanks.

**Comment by emmartins** [comment]
@xstefank @nziakova correct

File: microprofile-metrics/README.adoc (hunk 1 above)
Comment (emmartins) [reply microprofile-metrics/README.adoc:101]: missed this, fixed by https://github.com/wildfly/quickstart/pull/419

**Pull request merged**